runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -210,7 +213,7 @@ public static String htmlLabel(String key, int value)
 	{
 		String valueStr = value + "";
 
-		if (value > 9999999 || value < -9999999)
+		if (value > MAX_EXPERIENCE || value < MIN_EXPERIENCE)
```

Comment:
The check probably do not even needs to be here at all.

Reply:
I agree, I kept it because of the current implementation but it doesn't seem necessary so I have removed it.